dccrg.hpp

```diff
@@ -8927,12 +8932,14 @@ template <
 			return;
 		}
 
-		this->neighbors_of.at(cell) = this->find_neighbors_of(cell, this->neighborhood_of, this->max_ref_lvl_diff);
+		this->neighbors_of[cell] = this->find_neighbors_of(cell, this->neighborhood_of, this->max_ref_lvl_diff);
 		std::vector<uint64_t> found_neighbors_of;
-		for (const auto& i: this->neighbors_of.at(cell)) {
+		for (const auto& i: this->neighbors_of[cell]) {
 			found_neighbors_of.push_back(i.first);
 		}
-		this->neighbors_to.at(cell) = this->find_neighbors_to(cell, found_neighbors_of);
+		this->neighbors_to[cell] = this->find_neighbors_to(cell, found_neighbors_of);
```

Comment:
Can you please comment, why have you disabled range checks here?

Reply:
Is it not superfluous in assignment?

Reply:
Not if something is already supposed to be there I think